js/client-teambuilder.js

```diff
@@ -200,11 +235,33 @@
 			Storage.nwLoadTeams();
 		},
 		edit: function(i) {
+			if (this.curmoving !== false) return;
 			var i = +i;
 			this.curTeam = teams[i];
 			this.curTeamIndex = i;
 			this.update();
 		},
+		"move": function(i) {
+			if (this.curmoving !== false) return;
```

Comment:
In that case you probably want to set it to `null` as it's default state so it doesn't appear to be a Boolean. You might even want to compare to `if (this.curmoving >= 0) return;` if the position can be 0 or just `if (this.curmoving) return;` otherwise.

On another style note, to match the casing used with other variables you probably want the variable to be named `curMoving`.